src/parser/jobs/drg/modules/Positionals.js

```diff
@@ -0,0 +1,244 @@
+import {t} from '@lingui/macro'
+import {Trans, Plural} from '@lingui/react'
+import React, {Fragment} from 'react'
+import Module from 'parser/core/Module'
+import STATUSES from 'data/STATUSES'
+import ACTIONS from 'data/ACTIONS'
+import {getDataBy} from 'data'
+import {ActionLink} from 'components/ui/DbLink'
+import {Table, Message, Icon} from 'semantic-ui-react'
+import {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+
+const ROTATION_IDS = [
+	ACTIONS.RAIDEN_THRUST.id,
+	ACTIONS.DISEMBOWEL.id,
+	ACTIONS.CHAOS_THRUST.id,
+	ACTIONS.TRUE_THRUST.id,
+	ACTIONS.VORPAL_THRUST.id,
+	ACTIONS.FULL_THRUST.id,
+]
+
+const PROC_IDS = [ACTIONS.FANG_AND_CLAW.id, ACTIONS.WHEELING_THRUST.id]
+const NEXT_COMBO = {
+	[ACTIONS.FANG_AND_CLAW.id]: ACTIONS.WHEELING_THRUST.id,
+	[ACTIONS.WHEELING_THRUST.id]: ACTIONS.FANG_AND_CLAW.id,
+}
+
+const TRUE_NORTH_CHARGES = 2
+const TRUE_NORTH_CD = ACTIONS.TRUE_NORTH.cooldown * 1000
+
+export default class Positionals extends Module {
+	static handle = 'positionals'
+	static title = t('drg.positionals.title')`Positionals`
+	static dependencies = [
+		'jumps',
+		'combatants',
+		'suggestions',
+	]
+
+	// tracking raiden thrust procs
+	_rtCombos = []
+	_currentCombo = null
+
+	// true north tracking
+	_tnCharges = 2
```

Comment:
I've been rolling ideas around for a while now for how to implement a core solution for charge tracking. For the time being, doing it manually here is fine, but down the line it may be something to refactor.